packages/remark-parse/lib/tokenize/text.js

```diff
@@ -58,7 +56,7 @@ function text(eat, value, silent) {
   subvalue = value.slice(0, min);
   now = eat.now();
 
-  self.decode(subvalue, now, function (content, position, source) {
```

Comment:
I'd hold off on this sort of change unless it provides a benefit like avoiding `const that = this`. There is a perf cost to making a bunch of bound functions, though small.